harvard-cranfield-university.csl

```diff
@@ -270,25 +257,35 @@
             </group>
             <text variable="archive"/>
             <text variable="archive_location"/>
-            <text macro="online"/>
             <text macro="publisher"/>
           </group>
         </else-if>
-        <else-if type="article-journal article-magazine article-newspaper broadcast interview manuscript map patent personal_communication song speech thesis webpage motion_picture" match="any">
-          <group suffix=".">
+        <else-if type="article-journal" match="any">
+          <group delimiter=", ">
+            <text macro="title" prefix="'" suffix="'"/>
```

Comment:
Setting quotes="true" now gives me double quotes, and I need to specify punctuation-in-quote="false" otherwise the following comma gets swallowed in the quotes (as it is rendered in Mendeley). Are these locale specific (this is "en-GB")? Is there a way to use quotes="true" and get single quotes?